### ISSUE TITLE
Dns46 translator over internet

### ISSUE THREAD

**Issue by alireza-pmahdavi** [vitlabuda/get4for6#1]
Can I use this project to run a DNS46 server over the internet? The number of clients is less than 5000, and I also have static IPv4 and IPv6 addresses

**Comment by vitlabuda** [comment]
Yes, you can; after all, the [LICENSE](../blob/cdc618dc16d6cd1308648454e9ffc8d9e0d1f30d/LICENSE) does not prohibit it (also, the license says that I won't be taking any responsibility if anything goes wrong). However, I would recommend reading the README and the example config file thorougly, especially the ["security considerations" part](../blob/cdc618dc16d6cd1308648454e9ffc8d9e0d1f30d/get4for6.example.toml#L118-L134), before deploying it so that you know what to expect.
